components/Header.tsx

```
@@ -3,571 +3,248 @@
-                        <DropdownItem
-                            href="/gcl/professional-cloud-network-engineer-step-by-step"
-                            label="ステップバイステップ ガイド"
-                            desc="PCNE 完全攻略ガイド"
-                            ariaLabel="PCNE ステップバイステップ完全攻略ガイド"
-                            onClick={() => setOpenMenu(null)}
-                        />
-                    </div>
-                </div>
-
-                {/* Associate Google Workspace Administrator */}
+                    <svg width="18" height="18" viewBox="0 0 18 18" fill="none" aria-hidden>
```

Biome (2.4.15)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)